README.md

```diff
@@ -131,6 +131,27 @@ Build USBIP tools:
 /usr/src/4.19.43-microsoft-standard/tools/usb/usbip$ sudo make install -j 12
 ```
 
```

Comment:
Hi! Thanks for the commit. Q: Is this compiler message occurring because of an upgrade to GCC v8? Any ideas on why it did not occur before?

Reply:
Hi, thanks for replying! First of all, my kernel version is `4.19.104` on Ubuntu 20 with gcc `9.3.0`. The USB/IP Makefiles seem to be settings the `-Werror` flag, so all of these small warnings will make the build fail. These errors are new [features](https://gcc.gnu.org/gcc-9/changes.html#c-family) so that could be why they haven't shown up before. I updated the README to reflect that as well.

Reply:
I encountered the same error and these patches have resolved it (at least as far as being able to compile).